modules/consensus/accept_test.go

```diff
@@ -650,17 +639,15 @@ func (cst *consensusSetTester) testFileContractsBlocks() error {
 		}},
 	}
 	copy(txn.StorageProofs[0].Segment[:], segment)
-	err = cst.tpool.AcceptTransaction(txn)
+	err = cst.tpool.AcceptTransactionSet([]types.Transaction{txn})
```

Comment:
probably frequent enough

I worry about clarity though. If you make it a variadic function, I feel like the connotation is that the objects are not necessarily fundamentally related. But here, the objects are part of a set, and must have a specific order.